merch-dropper/src/utils/axiosWithEnv.js

```diff
@@ -1,14 +1,14 @@
-import axios from 'axios';
+import axios from "axios";
 
 export const axiosWithEnv = () => {
-    let url;
-    if (process.env.REACT_APP_BASE_URL === "development") {
-      url = "http://localhost:5032";
-    } else {
-      url = "https://merch-dropper.herokuapp.com";
-    }
-  
-    return axios.create({
-      baseURL: url,
-    });
-  };
\ No newline at end of file
+  let url;
+  if (process.env.REACT_APP_BASE_URL === "development") {
+    url = "https://merch-dropper.herokuapp.com";
```

Comment:
Same, why changing it?  If we change it to both being the same, what is the point of having this function?